crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -457,6 +520,34 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn test_set_transfer_batch_size_in_bytes_to_none() {
+        let temp_dir = tempfile::tempdir().unwrap();
+        let metadata_manager = create_metadata_manager(temp_dir.path()).await;
+
+        create_compressed_file(metadata_manager.clone(), temp_dir.path()).await;
+        create_compressed_file(metadata_manager.clone(), temp_dir.path()).await;
+
+        let (_, mut data_transfer) =
+            create_data_transfer_component(metadata_manager, temp_dir.path()).await;
+
+        data_transfer
+            .set_transfer_batch_size_in_bytes(None)
+            .await
+            .unwrap();
+
+        assert_eq!(data_transfer.transfer_batch_size_in_bytes, None);
```

Comment:
Maybe check the value is not `None` the call to `set_transfer_batch_size_in_bytes(None)`.

Reply:
Fixed.